doc/notebooks/solar_potential.ipynb

```diff
@@ -45,7 +45,7 @@
     "session = gj.init(\"../../data/grassdata\", \"nc_basic_spm_grass7\", \"user1\")\n",
     "\n",
     "# Set computational region to elevation raster\n",
-    "gs.run_command(\"g.region\", raster=\"elevation@PERMANENT\", flags=\"pg\")"
+    "gs.parse_command(\"g.region\", raster=\"elevation@PERMANENT\", flags=\"pg\")"
```

Comment:
The change is only relevant here and in viewshed_analysis.ipynb. Both are probably copy-pastes from some other stuff.